tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"testing"
+	"time"
+)
+
+func TestE2E(t *testing.T) {
+	aws, err := cloudprovider.CloudProviderFactory("/tmp/kubeconfig", "/tmp/awscredentials", "default",
```

Comment:
You need to clearly state your intentions here. You assume the files to be present at certain locations during your e2e run. Either you need to create them(by copying from source) or generate them on the fly. You cannot assume that `/tmp/kubeconfig` exists.